features/step_definitions/logging.rb

```diff
@@ -309,3 +309,11 @@
     raise "Get metrics failed with error, #{@result[:response]}"
   end
 end
+
+Given /^default storageclass is stored in the#{OPT_SYM} clipboard$/ do | cb_name |
```

Comment:
@QiaolingTang perhaps this step belongs to `storage_class.rb` instead since it's `storageclass` related step

Reply:
@pruan-rht OK, thank you.